GVFS/GVFS.UnitTests/Virtualization/Projection/SortedFolderEntriesTests.cs

```diff
@@ -166,7 +166,7 @@ private static void AddFiles(SortedFolderEntries entries, params string[] names)
             for (int i = 0; i < names.Length; i++)
             {
                 LazyUTF8String entryString = ConstructLazyUTF8String(names[i]);
-                entries.AddFile(entryString, new byte[20]);
+                entries.AddFile(entryString, new byte[20], isIncluded: true);

```

Comment:
Are there any unit tests that should be added where `isIncluded` is false?  (e.g. validating that `isIncluded` is properly set by `AddFile`)